### ISSUE TITLE
Microsoft.CodeAnalysis 1.3.1 & 1.3.2

### ISSUE THREAD

**Issue by dtkujawski** [DotNetAnalyzers/StyleCopAnalyzers#2205]
I was testing this code on an ASPNET CORE 1.0.1 project and it worked great.  However, I noticed that when I upgraded from 1.2.0 to 1.3.1 or 1.3.2 it stopped working with an error int he ASPNET CORE project saying it could find Microsoft.CodeAnalysis assemblies (v1.3.1).

I've also tried with 2.0.0-beta3 and same behavior, gives an error saying it can't find Microsoft.CodeAnalysis (2.0.0).

Has anyone else noticed this behavior?  The highest version that works for me is 1.3.0 when doing analysis on an ASPNET CORE 1.0.1 project.

**Comment by jimmymain** [comment]
I maintain a small documentation generation project [documentation.analysers](https://github.com/jimmymain/Documentation.Analyzers). I tried to upgrade all the nuget packages recently in an attempt to get my analysers working on .net core cli. **I have had the exact same problem**, I have gone back to using 1.0.0. 

I briefly managed to get the analysers working in a debug session by manually copying the libraries into the working directory, alas the immutable collections classes could not be loaded, manually copying those resulted in an instantiation error. `get_SupportedDiagnostics` has no implementation. I have spent many hours trying to resolve it. Any suggestions would be most welcome. How the libraries could be so broken as to load the analysers and then complain at runtime about unimplemented properties when the code compiles is alarming.

Its very disappointing that the tooling and support on .net core is so lacking. It's my understanding that none of the quick fixes work on .net core. I am thinking about building the stylecop.analyser vsix and installing it that way. 

At least that way I suspect I will get the quick fixes.

**Comment by dtkujawski** [comment]
I was able to successfully do code generation on my CORE 1.0.1 project, but there were several things that I had to keep in mind to get everything wired up:
- Analysis version has to be 1.3.0 or earlier
- Analysis project has to be portable

Beyond that, since .NET Standard 1.x (portable) doesn't support File I/O and other critical API's, I had to do some gymnastics to get around those restrictions.  But, in the end, it works and I have all my previous code generators which were written in CompileModules working now rewritten as Analyzers.

My question still remains, though - can't use the latest analysis packages or they fail to load the assemblies.
